### PR TITLE
[AM][OPS-10131] add message on POST response when DES resubmit queued

### PR DESCRIPTION
**Pull request by almorcrette**

### CODE REVIEW

File: app/uk/gov/hmrc/timetopay/arrangement/services/TTPArrangementService.scala [hunk 1]
```diff
@@ -80,11 +79,11 @@ class TTPArrangementService @Inject() (
                   AuditService.auditTags
                 )
 
-                returnedError
+                Future.failed(DesApiException(error.code, error.message, queuedForRetry = true))
```
Comment (andy1138) [inline app/uk/gov/hmrc/timetopay/arrangement/services/TTPArrangementService.scala]: keeping with the spirit of the old code you could have done 
`def returnedError(queuedForRetry:Boolean) = Future.failed(DesApiException(error.code, error.message), queuedForRetry)`
and used  
`returnedError(queuedForRetry = true)`

(but more up to you than wrong/right)

File: test/uk/gov/hmrc/timetopay/arrangement/controllers/TTPArrangementControllerSpec.scala [hunk 2]
```diff
@@ -70,7 +70,21 @@ class TTPArrangementControllerSpec extends ITSpec with TestData {
 
     result.status shouldBe 500
     result.body should include("Submission to DES failed, status code [400]")
```
Comment (andy1138) [inline test/uk/gov/hmrc/timetopay/arrangement/controllers/TTPArrangementControllerSpec.scala:72]: `status code [400]` not 500 ?

Comment (almorcrette) [reply test/uk/gov/hmrc/timetopay/arrangement/controllers/TTPArrangementControllerSpec.scala:72]: Well that's very weird. I'll look into it.

File: app/uk/gov/hmrc/timetopay/arrangement/services/TTPArrangementService.scala (hunk 1 above)
Comment (almorcrette) [reply app/uk/gov/hmrc/timetopay/arrangement/services/TTPArrangementService.scala]: I like the spirit of old code. will change

File: test/uk/gov/hmrc/timetopay/arrangement/controllers/TTPArrangementControllerSpec.scala (hunk 2 above)
Comment (almorcrette) [reply test/uk/gov/hmrc/timetopay/arrangement/controllers/TTPArrangementControllerSpec.scala:72]: I see, the mock was wrong

Comment (almorcrette) [reply test/uk/gov/hmrc/timetopay/arrangement/controllers/TTPArrangementControllerSpec.scala:72]: Have looked into this and confirm this is right.

the message with the 400 code refers to the DES response code. The 500 is what our ttp-arrangement service responds to our front end.

**Pull request merged**